automated_detection_testing/detection_service.py

```diff
@@ -79,13 +80,14 @@ def main(args):
     with open(ssh_key_name, "w") as ssh_key:
         ssh_key.write(response['KeyMaterial'])
     os.chmod(ssh_key_name, 0o600)
+    private_key_path = str(os.getcwd() + "/" + ssh_key_name)
 
     # build new version of ESCU
     sys.path.append(os.path.join(os.getcwd(),'security_content/bin'))
+
     try:
         module = __import__('generate')
-        module.sys.argv = ['generate', '-p', 'security_content', '-o' 'security_content/package']
-        results = module.main(module.sys.argv)
+        results = module.main(REPO_PATH = 'security_content' , OUTPUT_PATH = 'security_content/package', VERBOSE = '--verbose' )
```

Comment:
@patel-bhavin, I think the correct value for VERBOSe is either True or False. Maybe we should go for TRUE.

Reply:
Updated! Thanks for reviewing this

Reply:
Another small thing: The True needs to be without ' otherwise it's a string and not a boolean.

Reply:
hmmn, weird. it still worked for me and gives a verbose output